### PR TITLE
fix: correct url encoding for ga provider

### PR DESCRIPTION
**Pull request by iranagame**
Fix issue #25

### CODE REVIEW

File: src/runtime/providers/vercel.ts [hunk 1]
```diff
@@ -5,7 +5,7 @@ export interface Options {
   debug: boolean
 }
 
-const vitalsUrl = 'https://vitals.vercel-analytics.com/v1/vitals'
+const vitalsUrl = 'https://vitals.vercel-analytics.com/v1/vitals?d=1'
```
Comment (danielroe) [inline src/runtime/providers/vercel.ts:8]: I'm not sure there's any need to modify this URL - is there?
```suggestion
const vitalsUrl = 'https://vitals.vercel-analytics.com/v1/vitals'
```

Comment (iranagame) [reply src/runtime/providers/vercel.ts:8]: Yes, But this is just a dummy GET parameter at start to be compatible with `&` symbol on `encodeParams(...)` in `src/runtime/utils.ts`.

Comment (danielroe) [reply src/runtime/providers/vercel.ts:8]: Ah, understood. `encodeParams` is not used in this runtime provider, however, so I think we can safely omit this change.

Comment (iranagame) [reply src/runtime/providers/vercel.ts:8]: I see, you are right.

**Comment by iranagame** [comment]
Any chance to merge this?

**Review by danielroe — APPROVED** [review]

File: src/runtime/utils.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ export function encodeParams (obj) {
   let str = ''
   for (const k in obj) {
     if (obj[k]) {
-      str += `${k}=${encodeURIComponent(obj[k])}`
+      str += `&${k}=${encodeURIComponent(obj[k])}`
```
Comment (pi0) [inline src/runtime/utils.ts:17]: Refactor: We might use ufo to prevent similar issues in the future

**Review by pi0 — APPROVED** [review]

**Comment by iranagame** [comment]
Since its critical bug, is it also possible to merge this PR with chore(0.1.4) release?

**Pull request merged**